standalone-metastore/metastore-common/src/test/java/org/apache/hadoop/hive/common/metrics/metrics2/TestCodahaleReportersConf.java

```diff
@@ -149,4 +148,30 @@ public void testNoFallbackOnIncorrectConf() throws Exception {
 
     Assert.assertFalse(jsonReportFile.exists());
   }
+
+  /**
+   * Tests if MetricsFactory is initialised properly, if in case Metrics2Reporter is tried to add more than once.
+   *
+   * **/
+  @Test
+  public void testMetricsFactoryInitMetrics2ReporterAddedTwice() throws Exception {
+    if(MetricsFactory.getInstance() != null) {
+      MetricsFactory.close();
+    }
+    Configuration conf = new Configuration();
+
+    jsonReportFile = File.createTempFile("TestCodahaleMetrics", ".json");
+    System.out.println("Json metrics saved in " + jsonReportFile.getAbsolutePath());
```

Comment:
Can we use logger here rather than system out

Reply:
Sure